Module-6-Assignment/StudentInsertionTriggerTest.cls

```diff
@@ -0,0 +1,79 @@
+/**
+* Test class for testing StudentInsertionTrigger.
+*/
+@isTest
+public class StudentInsertionTriggerTest 
```

Comment:
Add test for MyCount where student class is changes

Reply:
Done.
Changed assertion of NumberOfStudents Field to My Count